draft-ietf-quic-qpack.md

```diff
@@ -1032,7 +1033,7 @@ the Base.
 ~~~~~~~~~~
 {: title="Indexed Field Line"}
 
-This representation starts with the '1' 1-bit pattern, followed by the 'T' bit
+This representation starts with the '1' one-bit pattern, followed by the 'T' bit
 indicating whether the reference is into the static or dynamic table.  The 6-bit
```

Comment:
For consistency?

```suggestion
indicating whether the reference is into the static or dynamic table.  The six-bit
```

Reply:
I should have mentioned this in the summary.  This is intentional, because the spec defines an "N-bit prefix" to mean something very specific, and that was consistently used in numeric format.  HPACK also had the "one-bit pattern"/"4-bit prefix" as well.  It just got out of sync in some of the QPACK additions.

Reply:
I agree with Ian that it would be nice to have a consistent rule across pattern and prefix about writing number numbers with digits versus words.  For example, always write with digits, or always write out in words.  I've also seen style guides recommending words for numbers up to ten and digits starting from 11 (just like in this sentence).  Well, N is always at most 7, so we should probably always use digits or always use words, but preferably do the same for patterns and prefixes within a single document.

I agree with Alan that consistency with HPACK is also desirable.  Now HPACK has 5 occurrences of "N-bit pattern" and 14 occurrences of "N-bit prefix" with specific values for N, and every single one of them uses digits to represent N (just like in this sentence).

In contrast, QPACK before this change has 10 occurrences of N-bit pattern with N in words, 2 occurrences of N-bit pattern with N in digits, 0 occurrences of N-bit prefix with N in words, and 20 occurrences of N-bit prefix with N in digits (only counting occurrences where N is a specific number, not A or N or N-1).  This change would make all 12 patterns written in words but leave all 20 prefixes in digits.  For consistency, I suggest changing all N-bit patterns to writing N in digits.

Reply:
@bencebeky : derp, you are absolutely right.  I know I looked at HPACK for consistency but in my head saw it was spelled out in some cases -- must be moving too fast.  I'll update it to be digits everywhere, which is both self-consistent in this document consistent with HPACK.